doc/source/install.rst

```diff
@@ -122,6 +122,10 @@ options you want to change and configure:
 |                     | if you use the Redis storage driver.              |
 +---------------------+---------------------------------------------------+
 
+The same options are also available as `incoming.<drivername>_` for configuring
```

Comment:
sure, yeah, i thought it just a bad link or something (and i didn't realise we prefix everything). seems best to follow format above and add _*